drivers/bus/acpi_new/uacpiosl.c

```diff
@@ -0,0 +1,375 @@
+#include "precomp.h"
+//#define NDEBUG
+#include <debug.h>
+
+UINT32
+ACPIInitUACPI()
+{
+    uacpi_status status = uacpi_initialize(0);
+    if (uacpi_unlikely_error(status)) {
```

Comment:
```suggestion
    if (uacpi_unlikely_error(status))
    {
```
Or simply remove the braces.